standalone/pruntime/enclave/src/system/gk.rs

```diff
@@ -230,13 +236,13 @@ where
                 .on_block_processed(self.block, &mut tracker);
 
             if worker_info.state.mining_state.is_none() {
-                // Mining already stopped, do nothing.
+                debug!("Mining already stopped, do nothing.");
```

Comment:
I think we need to add the worker pubkey to the log as well. GK maintain all the nodes together. We need to distinguish them

Reply:
Added it to most logs.